play/binary-explorer/index.html

```diff
@@ -64,6 +64,11 @@ <h1 class="text-center mt-2">
 			<p style="text-align: right">
 				<button class="btn btn-success" onclick="addRow()">Add Letter</button>    
 			</p>
+			<p style="text-align: right">
```

Comment:
These are more things of preference, but here are some User Interface things to consider:

- Should the buttons be displayed side by side, or on newlines as they are now?
- Should the buttons be different colors or not?
- Should the button text be changed to be more descriptive?
- Should the Add to Message button be disabled if there are no binary string rows on screen?